src/ReachSets/ContinuousPost/TMJets/post.jl

```diff
@@ -0,0 +1,68 @@
+using TaylorModels: validated_integ
+
+function post(𝒫::TMJets,
+              𝑆::AbstractSystem, # {<:ImplicitContinuousSystem}
+              invariant::Union{LazySet, Nothing},
+              𝑂::Options)::ReachSolution{Zonotope}
+
+    # ==================================
+    # Initialization
+    # ==================================
+
+    𝑂 = TwoLayerOptions(merge(𝑂, 𝒫.options.specified), 𝒫.options.defaults)
+
+    # system of ODEs
+    f! = 𝑆.s
+    n = 𝑂[:n]
+
+    # initial and final times, and maximum allowed number of steps
+    t0 = 0.0
+    T = 𝑂[:T]
+    max_steps = 𝑂[:max_steps]
+
+    # unrap algorithm-specific options
+    abs_tol, orderQ, orderT = 𝑂[:abs_tol], 𝑂[:orderQ] 𝑂[:orderT]
+
+    # initial sets
+    X0 = convert(IntervalBox, 𝑆.x0)
```

Comment:
Does this always work or do you need to call `box_approximation` first?

Reply:
It;s a good idea (more robust) to apply `box_approximation` first to `𝑆.x0` 👍 . 

all the benchmarks we tried so far had  hyperrectangles for the initial set :)